apis/placement/v1alpha1/override_types.go

```diff
@@ -56,6 +56,7 @@ type OverridePolicy struct {
 	// OverrideRules defines an array of override rules to be applied on the selected resources.
 	// The order of the rules determines the override order.
 	// When there are two rules selecting the same fields on the target cluster, the last one will win.
+	// There can only be one override rule if it is a `Delete` type override rule.
```

Comment:
there could be multiple override rules.

For example,

i want to override serviceExport on member-1, and delete serviceExport on member-2.